### PR TITLE
week2

### PR DESCRIPTION
**Pull request by mynewyear**

### CODE REVIEW

File: 01-node-tutorial/answers/writeWithPromisesAwait.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+const { writeFile, readFile } = require("fs").promises;
+
+
+const writer = async () => {
+    try { await writeFile(
```
Comment (zjpetersen) [inline 01-node-tutorial/answers/writeWithPromisesAwait.js:5]: There are some formatting issues here which make the file a bit hard to read.  In VSCode you can right click in the file and select 'Format Document' and it'll fix the indentation for you!

File: 01-node-tutorial/answers/writeWithPromisesAwait.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        './temporary/temp.txt',
```
Comment (zjpetersen) [inline 01-node-tutorial/answers/writeWithPromisesAwait.js:6]: Try to call writeFile three times, you'll see the lines append in the correct order thanks to the advantage of the 'await' keyword.

**Review by zjpetersen — APPROVED** [review]

File: 01-node-tutorial/answers/writeWithPromisesAwait.js (hunk 1 above)
Comment (mynewyear) [reply 01-node-tutorial/answers/writeWithPromisesAwait.js:5]: done

File: 01-node-tutorial/answers/writeWithPromisesAwait.js (hunk 2 above)
Comment (mynewyear) [reply 01-node-tutorial/answers/writeWithPromisesAwait.js:6]: done

**Pull request merged**